InstanceExport/PowerShell/InstanceExport.ps1

```diff
@@ -178,71 +186,87 @@ This function exports an given instancce and save it to your disk in the specifi
                 $fileName = $entry.FileName.Split([IO.Path]::GetInvalidFileNameChars()) -join ''
                 $filePath = "$directoryPath/$fileName"
                 $fileAlreadyExists = Test-Path -Path $filePath -PathType Leaf
-                if (!$fileAlreadyExists -or $overwrite)
-                {
+                if (!$fileAlreadyExists -or $overwrite) {
                     $entryExportParameters = @{
                         Method      = "GET"
                         Uri         = "https://$($manifestJson.SubDomain).$($manifestJson.HostName)$($entry.Url)"
                         Headers     = $header
                         ContentType = "application/json"  
                     }
                         
-                    try{
-                        if (!$entry.fileName.Contains(".json")){
+                    try {
+                        if (!$entry.fileName.Contains(".json")) {
                             Invoke-RestMethod @entryExportParameters -OutFile $filePath | Out-Null
                         }
-                        else{
+                        else {
                             $GetEntriesResponse = Invoke-RestMethod @entryExportParameters
                             $GetEntriesResponse | ConvertTo-Json -Depth 100 | Out-File -FilePath $filePath
                         }
                     }
-                    catch{
+                    catch {
                         $message = "Error occurred when extracting data from $($entry.Url)"
                         Log -msg "An error occurred" -displayMsg $message -logLevel "error" -currentDate $currentDate
                         Log -msg $message -logLevel "error" -currentDate $currentDate
                         Log -msg "StatusCode: $($_.Exception.Response.StatusCode.value__)" -logLevel "error" -currentDate $currentDate
                         Log -msg "Url: $($entry.Url)" -logLevel "error" -currentDate $currentDate
                         Log -msg "StatusDescription: $($_.Exception.Response.StatusDescription)" -logLevel "error" -currentDate $currentDate
+                        Log -msg "Exception: $($_.Exception)" -logLevel "error" -currentDate $currentDate
+                        Write-Verbose $_.Exception
+                        Write-Verbose $_.Exception.Response
+                        if ($_.Exception.Response.StatusCode -eq 503) {
+                            $message = "Server is offline. Waiting it to get back."
+                            Write-Verbose $message
+                            Log -msg $message -displayMsg $message -logLevel "info" -currentDate $currentDate
+                            do {
+                                $response = ServerHealtCheck("https://$($manifestJson.SubDomain).$($manifestJson.HostName)/en/healthcheck");
+								
+                            } while ($response.Exception)
+                            $message = "Server is back online"
+                            Write-Verbose $message
```

Comment:
If we let the user know that we stopped to wait we should let them know we're good to go again

```suggestion
                            $message = "Server is back online. Resuming export..."
                            Write-ColorOutput green $message
```

Reply:
...following on from my previous comment, I think "Server connection reestablished. Resuming export."

Reply:
I've updated with your considerations. It should be good for another review